akka-http-tests/src/test/scala/akka/http/scaladsl/server/EntityStreamingSpec.scala

```diff
@@ -190,6 +190,74 @@ class EntityStreamingSpec extends RoutingSpec with ScalaFutures {
     }
   }
 
+  "opaque-marshaller-example" in {
```

Comment:
Did that one fail before?

Reply:
Yup (I wrote the test first and it was failing).

Reply:
Great, thanks!